.gitignore

```
@@ -5,6 +5,10 @@
 # Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio, WebStorm and Rider
 # Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
 
+#DB File
+*Db.*
+*.db
+
 # User-specific stuff

```

```suggestion
.idea/
```

`.idea/`로 축약해서 작성하면 더 좋을 것 같습니다.